recipe/download_osx_sdk.sh

```diff
@@ -26,24 +26,35 @@ export CONDA_BUILD_SYSROOT="${OSX_SDK_DIR}/MacOSX${MACOSX_SDK_VERSION}.sdk"
 if [[ ! -d ${CONDA_BUILD_SYSROOT} ]]; then
```

Comment:
We should test this here; you could manually set a few different SDK versions in the variant configs
```suggestion
if [[ 1 == 1 ]]; then
```

Reply:
Looks like it's working.